pyroomacoustics/room.py

```diff
@@ -904,14 +871,23 @@ def _var_init(
             self.set_air_absorption()
 
         # default values for ray tracing parameters
-        self._set_ray_tracing_options(use_ray_tracing=ray_tracing)
+        self.set_ray_tracing()
+        if not ray_tracing:
+            self.unset_ray_tracing()
 
         # in the beginning, nothing has been
         self.visibility = None
 
         # initialize the attribute for the impulse responses
         self.rir = None
 
+        self.sh_deg = 12
+        self.print_filter = 0
+
+        self.DIRPAT_source_files=["LSPs_HATS_GuitarCabinets_Akustikmessplatz.sofa","Debug_sofa_file_source_2.sofa"]
```

Comment:
Maybe it would be better to have these variables imported from a dirpat specific file.